website/docs/cloud-docs/api-docs/private-registry/manage-module-versions.mdx

```diff
@@ -193,3 +193,82 @@ https://app.terraform.io/api/v2/organizations/hashicorp/registry-modules/private
   }
 }
 ```
+
+## Fetch data about a module version’s deprecation 
+
+The [private registry module API](/terraform/cloud-docs/api-docs/private-registry/modules#sample-registry-request-private-module) returns data about private registry modules, including that module’s deprecation status.
```

Comment:
```suggestion
Send a `GET` request to the `/modules/:GitHub-organization/:module/:provider/versions` endpoint to retrieve data about private registry modules, including the module's deprecation status. Refer to the [private registry module API example](/terraform/cloud-docs/api-docs/private-registry/modules#sample-registry-request-private-module) for additional information.
```
This is one of those instances where because the usage docs are intermingled with the reference docs the path to the information is a little wonky. Instead of linking to the endpoint reference  page right away, which could confuse people, you should state the endpoint first, then link to the reference ( I guess example in this case).